Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.9.0-alpine3.12
+
+RUN pip install --upgrade google-api-python-client google-auth-httplib2 google-auth-oauthlib
+
+COPY ./requirements.txt ./requirements.txt
+
+RUN pip install -r requirements.txt
+
+COPY ./config.json ./config.json
```

Comment:
This file does not exists (it's even git excluded)
Would it be a good idea to define a default file?

Reply:
Ok. How we can do that?

Reply:
This would be a solution:
```dockerfile
COPY config-example.json ./config.json
```

But not sure it's really a "default" config, neither a "default" config has any meaning in this context @Riduidel?

Reply:
Well, it's more an example config that it would be nice to have in image, alongside with a command allowing to generate a config file from this example.